concrete/src/Cache/Page/PageCache.php

```diff
@@ -152,7 +153,8 @@ public function shouldAddToCache(PageView $v)
             return false;
         }
 
-        if ($u->isRegistered() || $_SERVER['REQUEST_METHOD'] == 'POST') {
+        $request = $app->make(Request::class);
+        if ($u->isRegistered() || $request->getMethod() === $request::METHOD_POST) {
```

Comment:
This `POST` method check here strikes me as something that should happen elsewhere. Maybe before trying to use the page cache we should just check if the request is a `POST`, then we don't need to juggle this request object in here

Reply:
Well, this method is called `shouldAddToCache` and it performs all the checks required to determine if a page should be placed in the cache. I'd prefer not to split these checks in two parts....